plugins/woocommerce/src/Blocks/BlockTypes/CustomerAccount.php

```diff
@@ -70,11 +70,10 @@ public function modify_hooked_block_attributes( $parsed_hooked_block, $hooked_bl
 		$parsed_hooked_block['attrs']['iconStyle']    = 'line';
 		$parsed_hooked_block['attrs']['iconClass']    = 'wc-block-customer-account__account-icon';
 
-		/*
-		* The Mini Cart block (which is hooked into the header) has a margin of 0.5em on the left side.
-		* We want to match that margin for the Customer Account block so it looks consistent.
-		*/
-		$parsed_hooked_block['attrs']['style']['spacing']['margin']['left'] = '0.5em';
```

Comment:
Should we remove it? 
@tjcafferkey could have more context on this.

Reply:
This provided some consistent spacing between icons which the user had control over via block styles rather than hard-coded values by CSS. The spacing, or lack thereof will be apparent when any focus or hover states are applied on the UI. 

Just curious, are we removing it here because we've moved this style elsewhere (e.g. in CSS) or because it is no longer needed at all?

Reply:
I was forced to remove this since it's not homogeneous around the whole icon, which means, this works good for horizontal headers only. But when rendering this block in a vertical manner, a visible misalignment appears. So we should avoid to do this by default as we should consider each block to behave in a generic manner.

A way to solve this could be wrapping both the customer account and the mini cart into a group block and this one should be the responsible of the alignment and separation of the children. So when in the context of the horizontal header, the group block can be presented as a row and when in the vertical header it can be presented as a stack. With that we can use the block structure an its attributes to maintain the visuals consistent without writing custom css or modifying the initial state of the child block.